src/components/App.js

```diff
@@ -1,26 +1,16 @@
 /* eslint-disable react/no-unused-prop-types */
 /* eslint-disable react/forbid-prop-types */
 import React from 'react';
-import PropTypes from 'prop-types';
-import { connect } from 'react-redux';
```

Comment:
Do you need to disable prop-types here? 🤔 Kindly remove the disabled rule, it seems to be redundant as you haven't used any props in your components here.